COPY/etc/manageiq/postgresql.conf.d/01_miq_overrides.conf

```diff
@@ -51,6 +56,9 @@ hot_standby = on
 # QUERY TUNING
 #------------------------------------------------------------------------------
 
+random_page_cost = 4
+effective_cache_size = 6GB
+default_statistics_target = 100
```

Comment:
This is the default so we can remove this.

Reply:
👍